Tools/wasm/wasm_build.py

```diff
@@ -198,12 +214,24 @@ def _check_emscripten():
         version = version[:-4]
     version_tuple = tuple(int(v) for v in version.split("."))
     if version_tuple < EMSDK_MIN_VERSION:
-        raise MissingDependency(
+        raise ConditionError(
             os.fspath(version_txt),
             f"Emscripten SDK {version} in '{EMSCRIPTEN_ROOT}' is older than "
             "minimum required version "
             f"{'.'.join(str(v) for v in EMSDK_MIN_VERSION)}.",
         )
+    broken = EMSDK_BROKEN_VERSION.get(version_tuple)
+    if broken is not None:
+        raise ConditionError(
+            os.fspath(version_txt),
+            f"Emscripten SDK {version} in '{EMSCRIPTEN_ROOT}' has known ",
+            f"bugs, see {broken}.",
```

Comment:
I think Black inserted a comma you didn't want.

```suggestion
            (f"Emscripten SDK {version} in '{EMSCRIPTEN_ROOT}' has known "
            f"bugs, see {broken}."),
```